cfme/services/catalogs/orchestration_template.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 import attr
+from miq_version import LOWEST
```

Comment:
**Suggestion:** 
To have uniformity, I would suggest to use `from cfme.utils.version import LOWEST`. I can see in the many test files we have this kinda implementation.